code/controllers/subsystem/atoms.dm

```diff
@@ -23,7 +23,7 @@ SUBSYSTEM_DEF(atoms)
 
 
 
-/datum/controller/subsystem/atoms/proc/InitializeAtoms(list/atoms)
+/datum/controller/subsystem/atoms/proc/InitializeAtoms(list/atoms, noisy = TRUE)
```

Comment:
This is one of the places I feel most uncomfortable.
How would I go about resuming atom init, with late init handling, without changes such as these?
I'm also noticing the `initialized` variable being modified heavily here - what does the value of that flag mean? If the flag is being modified here, what's the point of the `map_loader_` procs? How would I be able to get at the `late_loaders` list that'd be filled in by just doing `SSatoms.InitAtom()` in a loop?